tests_config/run_spec.js

```diff
@@ -144,17 +142,12 @@ global.run_spec = (dirname, parsers, options) => {
       typeof rangeStart === "undefined" &&
       typeof rangeEnd === "undefined" &&
       typeof cursorOffset === "undefined" &&
-      !TEST_CRLF
+      !TEST_CRLF &&
+      !unstableTests.has(filename)
     ) {
       test(`${filename} second format`, () => {
         const secondOutput = format(output, filename, mainOptions);
-        if (unstableTests.has(filename)) {
-          // To keep eye on failed tests, this assert never supposed to pass,
-          // if it fails, just remove the file from `unstableTests`
-          expect(secondOutput).not.toEqual(output);
```

Comment:
Seemed to be a good idea. Why have you decided to remove this?

Reply:
I forgot which file, but it formats twice with different options in `jsfmt.spec.js` run this twice, and one fail one pass, can't assert.

Reply:
I'll add it back

Reply:
Done.

Reply:
@thorn0 I think this is a better way to manage these todos, we should refactor `disableBabelTS` option to this way.

Reply:
#7786

Reply:
I'm not sure this is a better way. Why do you think so?

Reply:
I mean we can keep these assertions and still use an option to mark unstable/disabled/etc tests? Why clutter `run_spec.js` with these lists?

Reply:
> I'm not sure this is a better way. Why do you think so?

I thought they are todos